src/ui/public/filter_bar/lib/map_geo_polygon.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+
+export function FilterBarLibMapGeoPolygonProvider(Promise, courier) {
+  return function (filter) {
+    if (filter.geo_polygon) {
+      return courier
+      .indexPatterns
```

Comment:
I have to object here. I'm following the existing style of all the other files in this module. Changing just this one file makes it inconsistent and harder to read in context. We either need to revert this indentation rule (which was thoughtlessly changed) or fix the entire codebase in one go, otherwise it's just making things worse. I'll open a separate issue to discuss this.